politeiawww/api/v1/v1.go

```diff
@@ -184,19 +186,30 @@ const (
 	PropVoteStatusFinished      PropVoteStatusT = 4 // Proposal vote has been finished
 	PropVoteStatusDoesntExist   PropVoteStatusT = 5 // Proposal doesn't exist
 
-	// User edit actions
-	UserEditInvalid                         UserEditActionT = 0 // Invalid action type
-	UserEditExpireNewUserVerification       UserEditActionT = 1
-	UserEditExpireUpdateKeyVerification     UserEditActionT = 2
-	UserEditExpireResetPasswordVerification UserEditActionT = 3
-	UserEditClearUserPaywall                UserEditActionT = 4
-	UserEditUnlock                          UserEditActionT = 5
-	UserEditDeactivate                      UserEditActionT = 6
-	UserEditReactivate                      UserEditActionT = 7
+	// User manage actions
+	UserManageInvalid                         UserManageActionT = 0 // Invalid action type
+	UserManageExpireNewUserVerification       UserManageActionT = 1
+	UserManageExpireUpdateKeyVerification     UserManageActionT = 2
+	UserManageExpireResetPasswordVerification UserManageActionT = 3
+	UserManageClearUserPaywall                UserManageActionT = 4
+	UserManageUnlock                          UserManageActionT = 5
+	UserManageDeactivate                      UserManageActionT = 6
+	UserManageReactivate                      UserManageActionT = 7
 
 	// Authorize vote actions
 	AuthVoteActionAuthorize = "authorize" // Authorize a proposal vote
 	AuthVoteActionRevoke    = "revoke"    // Revoke a proposal vote authorization
+
```

Comment:
Just make these all consecutive. It'll make life easier If a human has to read it to debug.

Reply:
Done.